### PR TITLE
add VCF output/input to delly/call

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/delly/call/main.nf [hunk 1]
```diff
@@ -2,36 +2,40 @@ process DELLY_CALL {
...
     input:
     tuple val(meta), path(input), path(input_index), path(exclude_bed)
     path fasta
     path fai
 
     output:
-    tuple val(meta), path("*.bcf"), emit: bcf
-    tuple val(meta), path("*.csi"), emit: csi
-    path "versions.yml"           , emit: versions
+    tuple val(meta), path("*.{b,v}cf")  , emit: bcf
```
Comment (matthdsm) [inline modules/nf-core/delly/call/main.nf:16]:
```suggestion
    tuple val(meta), path("*.{bcf,vcf}")  , emit: bcf
```
I think this is clearer

**Review by matthdsm — APPROVED** [review]
LGTM, but don't merge until anyone else has had a say.

File: modules/nf-core/delly/call/main.nf (hunk 1 above)
Comment (nvnieuwk) [reply modules/nf-core/delly/call/main.nf:16]: Yes! Thanks!

**Review by peterpru — APPROVED** [review]

File: tests/modules/nf-core/delly/call/main.nf [hunk 2]
```diff
@@ -8,6 +8,8 @@ workflow test_delly_call_bam {
     input     = [ [ id:'test' ], // meta map
                     file(params.test_data['homo_sapiens']['illumina']['test_paired_end_recalibrated_sorted_bam'], checkIfExists: true),
                     file(params.test_data['homo_sapiens']['illumina']['test_paired_end_recalibrated_sorted_bam_bai'], checkIfExists: true),
+                    [],
```
[the remark this answers is not in the document]
Comment (nvnieuwk) [reply tests/modules/nf-core/delly/call/main.nf:11]: These are optional inputs. If we use `[]` it means this input is empty (currently there is no other way in nextflow to define optional inputs). Also thanks for pointing out the stub, I'll add it :)

**Pull request merged**